@xen-orchestra/lite/src/components/Console.tsx

```diff
@@ -78,7 +112,22 @@ const Console = withState<State, Props, Effects, Computed, ParentState, ParentEf
     },
   },
   ({ scale, state }) => (
-    <div ref={state.container} style={{ margin: 'auto', height: `${scale}%`, width: `${scale}%` }} />
+    <>
+      {state.rfb !== undefined && !state.rfbConnected && (
+        <p>
+          <FormattedMessage id='reconnectionAttempt' />
+        </p>
+      )}
+      <div
+        ref={state.container}
+        style={{
+          display: `${state.rfbConnected ? 'block' : 'none'}`,
```

Comment:
Why?

Reply:
When we try to connect to the console, a new `RFB` instance is created.
This instance generate a black square as console.
So when a `VM reboot`, a black square appear and disapear for each reconnection attempt.
This code is for show the console, only in the case we are connected to the console.

Reply:
The issue with `display: none` is that it will break the page flow if there are elements around the console because it takes no space at all.